server/app/durablejobs/RecurringJobExecutionTimeResolvers.java

```diff
@@ -55,4 +55,16 @@ public Instant resolveExecutionTime(Clock clock) {
           .toInstant();
     }
   }
+
+  /** Nightly at 2am local time */
+  public static final class Nightly2Am implements RecurringJobExecutionTimeResolver {
```

Comment:
suggestion (blocking): please add a test case to https://github.com/civiform/civiform/blob/main/server/test/durablejobs/RecurringJobExecutionTimeResolversTest.java#L10

Reply:
good call, this uncovered that it was setting it to run at 2am that same day (so always in the past). i bumped it forward a day so it's starting at 2am that night.

Reply:
Oh good 😅